Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -143,10 +141,10 @@ public struct Configuration: Equatable {
                 configPath = fullPath
                 return
             } else {
-                failIfRequired()
+                fail()
```

Comment:
@NachoSoto @jpsim That is now the [current behavior](https://github.com/realm/SwiftLint/blob/69477dab924ff61854c90903b30090ff74b799de/Source/SwiftLintFramework/Models/Configuration.swift#L107-L109).